apps/vscode-extension/src/extension.ts

```diff
@@ -24,8 +24,7 @@ export function activate(context: ExtensionContext) {
     commands.registerCommand('otter.generate.fixture', generateFixtureGenerateCommand(context)),
     commands.registerCommand('otter.menu.generate.fixture', generateFixtureGenerateCommand(context)),
     commands.registerTextEditorCommand('otter.extract.styling.variable', extractToVariable(context)),
-    commands.registerTextEditorCommand('otter.menu.extract.styling.variable', extractToVariable(context)),
```

Comment:
why is it removed?

Reply:
We found a way to specify Otter: Only in the Command Panel by adding a category so we do not need another command to have different names :)